packages/catalog-model/alpha-api-report.md

```diff
@@ -0,0 +1,34 @@
+## API Report File for "@backstage/catalog-model"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { JsonObject } from '@backstage/types';
+import { SerializedError } from '@backstage/errors';
+
+// Warning: (ae-forgotten-export) The symbol "Entity" needs to be exported by the entry point alpha.d.ts
+// Warning: (ae-unresolved-link) The @link reference could not be resolved: The package "@backstage/catalog-model" does not have an export "Entity"
```

Comment:
Kind of a feature actually, it needs an explicit export from `@backstage/catalog-model`, otherwise it exports the `Entity` type again. I think that's what we want tbh. Only problem is the import linter isn't quite clever enough to realize we're importing ourself and complains about a missing dep, but think we'll leave that one for later.